integration-tests/utils/integration-helpers.js

```diff
@@ -306,19 +482,38 @@ const gotoAnalysisTab = async (page, token, testUrl, workspaceName) => {
   await dismissNotifications(page);
 };
 
+// See src/analysis/utils/resource-utils.ts
+const isResourceDeletable = (resourceType, status) =>
+  ((lowStatus) => {
+    switch (resourceType) {
+      case 'runtime':
+        return ['unknown', 'running', 'updating', 'error', 'stopping', 'stopped', 'starting'].includes(lowStatus);
+      case 'app':
+        return ['unspecified', 'running', 'error'].includes(lowStatus);
+      case 'disk':
+        return ['failed', 'ready'].includes(lowStatus);
+      default:
+        console.error(`Cannot determine deletability; resource type ${resourceType} must be one of runtime, app or disk.`);
+        return undefined;
+    }
+  })(_.lowerCase(status));
+
 module.exports = {
   clickNavChildAndLoad,
   createEntityInWorkspace,
   defaultTimeout,
+  deleteAppsV2,
   deleteRuntimes,
   deleteRuntimesV2,
-  navigateToDataCatalog,
+  deleteWorkspaceV2,
```

Comment:
If there are no uses of `deleteWorkspaceV2`, let's not export it. Do we even need the const to be defined?

Reply:
I removed the const and reverted the exported method to the one wrapped by `withSignedInPage`